scaffolds/python/src/test/test_my_module.py

```diff
@@ -1,9 +1,11 @@
 import pytest
 from my_module import my_function
 
+
 def test_my_function():
     assert my_function(1, 3) == 4
 
+
```

Comment:
Was this line added on purpose ?

Reply:
Yes: you should have two blank lines between module functions (and one between class functions)

"Surround top-level function and class definitions with two blank lines." ( https://peps.python.org/pep-0008/#blank-lines )